SUMMARY.md

```diff
@@ -177,6 +177,7 @@
   - [Service Portal Education Student Assessment](libs/service-portal/education-student-assessment/README.md)
   - [Service Portal Eligibility](libs/service-portal/eligibility/README.md)
   - [Icelandic Names Registry](libs/service-portal/icelandic-names-registry/README.md)
+    - [Service Portal Settings Access Control](libs/service-portal/settings/access-control/README.md)
```

Comment:
It's missing a README.md at the root of the settings folder cf:
https://docs.devland.is/repository/gitbook-contribute#root-readme-md-within-handbook-directory

Reply:
Is that the general rule for parent directories?

Since the settings folder isn't a library, but a parent folder for multiple libraries. That way, we need to `nx` ignore it as well.

Reply:
Yes, it's the general rule. If the folder contains only libraries/apps then we add the root README to the .nxignore. It's afaik the only way for Gitbook to successfully create the default root page to have a link to each sub-README within, for example: https://docs.devland.is/technical-overview/x-road

Reply:
Makes sense 👍